packages/snap/src/index.ts

```diff
@@ -1,11 +1,37 @@
 import {
   type OnRpcRequestHandler,
+  type OnInstallHandler,
   MethodNotFoundError,
+  text,
+  panel,
 } from '@metamask/snaps-sdk';
 
 import { getPublicKey, signTransaction } from './rpc';
 import { AntelopeRequest, AntelopeSignatureRequest } from './types';
 
+const SNAP_NAME = 'Antelope MetaMask';
```

Comment:
This needs to be "EOS Wallet" I believe.

While the plugin itself is generally Antelope-compatible, the release I think we will do is EOS-focused, and to line up with the other Snaps, we can just call them "CHAIN Wallet".

I'm not sure if there are other places in the app that call it an "Antelope" product, but if there are - if we could make them all reference this variable (to easily rename the release), that'd be awesome

Reply:
Renamed the snap 👍 The other place where `Antelope` is used is in the package name which is currently `@greymass/antelope-snap`. I guess that should be `@greymass/eos-snap` as we'll probably need a different repo for each chain? 🤔

Reply:
Yeah, makes sense. I renamed the repo to: 

https://github.com/greymass/eos-wallet-snap

Yeah we may need a repo for each chain. Unsure of what the best structure is. We could:

- Clone this codebase a number of times, once for each chain
- Use branches on this repo, one branch for each chain
- Setup one core Antelope repo, and then make a fork for each chain

That last option could be the best? It would let us pull changes from upstream (the Antelope repo) into each chain repo, as changes become available.

Reply:
Yeah, I agree. In that case should we rename the repo to @greymass/antelope-snap?

Reply:
The one argument I can think of for using forks instead of branches would be more fine grained permissions, like if we wanted to be able to allow specific collaborators from specific chains.

Another argument might be around CI/CD, if we want to go that route 🤔

Reply:
The advantage of going the branch route is that we won't have to worry about having to explain why we have different packages to the consensys folks.

Reply:
We can always start with branches for now. If ever it ends up making collaborating with other teams more difficult then we can change our strategy at that point.

Reply:
> The advantage of going the branch route is that we won't have to worry about having to explain why we have different packages to the consensys folks.

They have to be different packages no matter what we do. Did you mean different repositories?

It also might be just as confusing to them to have one repo with different branches 😓

Reply:
Yeah sorry, I meant repositories. I think that sticking with the single repository that was already audited will be simpler than trying to submit a fork of a repository that was audited.

Reply:
I guess I'm not against submitting a fork and seeing if it works. I could just be overthinking this 😅

Reply:
In any case, we agree that we should rename this repo to `greymass/antelope-snap`? From there, we could either create a new branch or fork for each new network wallet.

Reply:
Yeah, I'll rename it back to what it was - and we can create a branch.

The audit also specifically says `greymass/antelope-snap`, and even renaming it might cause some confusion. 

The package we release however should be named `eos-wallet` (or similar) when we publish to NPM/etc.